int8 nNum
float32 fXlt
float32 fYlt
float32 fWidth
float32 fHeight
float32 fScore
